src/python_tests_runner/PythonTestRunner.cs

```diff
@@ -35,6 +35,7 @@ static IEnumerable<string[]> PythonTestCases()
             // Add the test that you want to debug here.
             yield return new[] { "test_indexer", "test_boolean_indexer" };
             yield return new[] { "test_delegate", "test_bool_delegate" };
+            yield return new[] { "test_subclass", "test_virtual_generic_method" };
```

Comment:
I see test failures. What happens if you drop this change? I could take a look at the crash later if we create a work item for it.